packages/commonwealth/server/migrations/20240320195632-create-Wallet-model.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('Wallets', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      user_id: {
```

Comment:
If we know what the query pattern for this schema will look like, are there indexes we should add e.g. `user_id` and/or `user_address`?

Reply:
Added an index on user_id. That will be how records are typically looked up